test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.CheckViewData.NullViewModel.html

```diff
@@ -0,0 +1,54 @@
+<div class="row">
+    <h4>At Model index</h4>
+    <div class="col-md-3">MetadataKind: 'Type'</div>
+    <div class="col-md-3">ModelType: 'ViewModel'</div>
+</div>
+
+<div class="row">
+    
+<h5>Template for ViewModel</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'ViewModel'</div>
+
+</div>
+<div class="row">
+    
+<h5>Partial for ViewModel</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'ViewModel'</div>
+
+</div>
+<div class="row">
+    <h5>Check View Data view component</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'ViewModel'</div>
+
+<h5>Check View Data view component's view</h5>
+<div class="col-md-3">MetadataKind: 'Type'</div>
+<div class="col-md-3">ModelType: 'ViewModel'</div>
+
+</div>
+<div class="row">
+    
+<h5>Template for Int32</h5>
+<div class="col-md-3">MetadataKind: 'Property'</div>
+<div class="col-md-3">ModelType: 'Int32'</div>
+    <div class="col-md-3">PropertyName: 'Integer'</div>
+
+</div>
+<div class="row">
+    
+<h5>Template for Int64</h5>
+<div class="col-md-3">MetadataKind: 'Property'</div>
+<div class="col-md-3">ModelType: 'Nullable`1'</div>
+    <div class="col-md-3">PropertyName: 'NullableLong'</div>
```

Comment:
Type is correct but all other property information lost w/o `src` changes. Instead get
``` html
<div class="col-md-3">MetadataKind: 'Type'</div>
<div class="col-md-3">ModelType: 'Nullable`1'</div>
```